lib/capistrano/tasks/migrations.rake

```diff
@@ -0,0 +1,169 @@
+          local_maintenance_path = "#{local_maintenance_path}/../../maintenance"
+
+          # Create a maintenance folder containing the index page from our gem
+          execute :mkdir, "#{shared_path}/maintenance"
+
+          # copy the contents of the index.html file to our shared folder
+          upload! File.open(local_maintenance_path + '/index.html'), "#{shared_path}/maintenance/index.html"
+
+          # copy the contents of the .htaccess file to our shared folder
+          upload! File.open(local_maintenance_path + '/.htaccess'), "#{shared_path}/maintenance/.htaccess"
+        end
+      end
+    end
+
+    desc 'fills in the executed_migrations on first deploy'
+    task :first_deploy do
+      on roles(:web) do
+        # Put all items in the migrations folder in the executed_migrations file
+        # When doing a deploy:setup, we expect the database to already contain
+        # The migrations (so a clean copy of the database should be available
+        # when doing a setup)
+        folders = capture("if [ -e #{release_path}/migrations ]; then ls -1 #{release_path}/migrations; fi").split(/\r?\n/)
+
+        folders.each do |dirname|
+          run "echo #{dirname} | tee -a #{shared_path}/executed_migrations"
+        end
+      end
+    end
+
+    desc 'runs the migrations'
+    task :execute do
+      on roles(:web) do
+        # If the current symlink doesn't exist yet, we're on a first deploy
+        currentDirectoryExists = capture("if [ ! -e #{current_path} ]; then echo 'yes'; fi").chomp
+        if currentDirectoryExists == 'yes'
+          invoke 'sumo:migrations:first_deploy'
+        end
+
+        # Check if there are new migrations found
+        folders = capture("if [ -e #{release_path}/migrations ]; then ls -1 #{release_path}/migrations; fi").split(/\r?\n/)
+
+        if folders.length > 0
+          executedMigrations = capture("cat #{shared_path}/executed_migrations").chomp.split(/\r?\n/)
+          migrationsToExecute = Array.new
+
+          # Fetch all migration directories that aren't executed yet
+          folders.each do |dirname|
+            migrationsToExecute.push(dirname) if executedMigrations.index(dirname) == nil
+          end
+
+          if migrationsToExecute.length > 0
+            # This can take a while and can go wrong. let's show a maintenance page
+            # and make sure we can put back the database
+            invoke 'sumo:migrations:symlink_maintenance'
+            invoke 'sumo:migrations:backup_database'
+
+            # run all migrations
+            migrationsToExecute.each do |dirname|
+              migrationpath = "#{release_path}/migrations/#{dirname}"
+              migrationFiles = capture("ls -1 #{migrationpath}").split(/\r?\n/)
+
+              migrationFiles.each do |filename|
+                if filename.index('locale.xml') != nil
+                  execute :php, "#{release_path}/bin/console forkcms:locale:import -f #{migrationpath}/#{filename}"
```

Comment:
This is fork specific, so I would PR the whole change to tijsverkoyen/capistrano-forkcms instead.